docker-compose.yml

```diff
@@ -43,17 +46,20 @@ services:
     platform: linux/amd64
     image: us-docker.pkg.dev/argus-labs/world-engine/relay/nakama@sha256:60737f1de75b5e1dfe0f1eb557ebf6f8c691cc2812950dc4e3132242709ddc09
     depends_on:
-      - postgres
-      - cardinal
+      cockroachdb:
+        condition: service_healthy
+      cardinal:
+        condition: service_started
     environment:
       - CARDINAL_ADDR=${CARDINAL_ADDR:-cardinal:3333}
       - CARDINAL_NAMESPACE=world
+      - DB_PASSWORD=${DB_PASSWORD:-development}
     entrypoint:
       - "/bin/sh"
       - "-ecx"
       - >
-        /nakama/nakama migrate up --database.address postgres:localdb@postgres:5432/nakama &&
-        exec /nakama/nakama --config /nakama/data/local.yml --database.address postgres:localdb@postgres:5432/nakama        
+        /nakama/nakama migrate up --database.address root:$DB_PASSWORD@cockroachdb:26257/nakama &&
+        exec /nakama/nakama --config /nakama/data/local.yml --database.address root:$DB_PASSWORD@cockroachdb:26257/nakama
```

Comment:
this doesn't use the fallback if DB_PASSWORD is not set?

Reply:
Default values for fallback are already managed in this section of service environment configuration, so there's no need to specify them again:
``` 
environment:
      - DB_PASSWORD=${DB_PASSWORD:-development}
```